MyMentorProject/src/main/java/com/Deny/MyMentorProject/User/User.java

```diff
@@ -0,0 +1,49 @@
+package com.Deny.MyMentorProject.User;
+
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class User {
+
+    private Long id;
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    private  String name;
```

Comment:
Format your code and define your variables first, then the constructors, then getters and setters, then equals and hash code and toString